crates/sui-core/src/authority.rs

```diff
@@ -1888,6 +1908,38 @@ impl AuthorityState {
         }
     }
 
+    pub async fn get_move_objects<T>(
+        &self,
+        owner: SuiAddress,
+        type_: &StructTag,
+    ) -> SuiResult<Vec<T>>
+    where
+        T: DeserializeOwned,
+    {
+        let object_ids = self
+            .get_owner_objects_iterator(owner)?
+            .filter(move |o| Self::matches_type(&ObjectType::Struct(type_.clone()), &o.type_))
+            .map(|info| info.object_id);
+        let mut staked_suis = vec![];
```

Comment:
```suggestion
        let mut move_objects = vec![];
```

You probably forgot to rename this after refactoring

Reply:
good catch!